Server/Plugins/APIDump/APIDesc.lua

```diff
@@ -2277,6 +2277,11 @@ return
 			},
 			Variables =
 			{
+				m_BlockHeight =
+				{
+					Type = "bool",
```

Comment:
Now I know what I could add in my plugin APIFuzzing checking the return type with the api doc :)